src/RazorConsole.Core/Vdom/Translators/HtmlSelectElementTranslator.cs

```diff
@@ -0,0 +1,145 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using RazorConsole.Core.Rendering.ComponentMarkup;
+using RazorConsole.Core.Vdom;
+using Spectre.Console;
+using Spectre.Console.Rendering;
+
+namespace RazorConsole.Core.Rendering.Vdom;
+
+public sealed class HtmlSelectElementTranslator : IVdomElementTranslator
+{
+    public int Priority => 80;
+
+    public bool TryTranslate(VNode node, TranslationContext context, out IRenderable? renderable)
+    {
+        renderable = null;
+
+        if (node.Kind != VNodeKind.Element || !string.Equals(node.TagName, "select", StringComparison.OrdinalIgnoreCase))
+        {
+            return false;
+        }
+
+        var options = ExtractOptions(node);
+        var selectedValue = VdomSpectreTranslator.GetAttribute(node, "value");
+        var placeholder = VdomSpectreTranslator.GetAttribute(node, "placeholder") ?? "Select an option";
+        var disabled = node.Attributes.ContainsKey("disabled");
+        var expand = VdomSpectreTranslator.TryGetBoolAttribute(node, "data-expand", out var expandValue) && expandValue;
+
+        // Determine display value and style
+        string displayValue;
+        string displayStyle;
+
+        if (!string.IsNullOrEmpty(selectedValue) && options.Any(o => string.Equals(o.Value, selectedValue, StringComparison.Ordinal)))
+        {
+            var selectedOption = options.First(o => string.Equals(o.Value, selectedValue, StringComparison.Ordinal));
+            displayValue = selectedOption.Label;
+            var colorAttr = VdomSpectreTranslator.GetAttribute(node, "data-value-color");
+            displayStyle = !string.IsNullOrWhiteSpace(colorAttr) ? colorAttr : "white";
+        }
+        else
+        {
+            displayValue = placeholder;
+            var colorAttr = VdomSpectreTranslator.GetAttribute(node, "data-placeholder-color");
+            var baseColor = !string.IsNullOrWhiteSpace(colorAttr) ? colorAttr : "grey70";
+            displayStyle = $"{baseColor} italic dim";
+        }
+
+        // Build the display markup
+        var markupText = ComponentMarkupUtilities.CreateStyledMarkup(displayStyle, displayValue, requiresEscape: true);
+
+        // Determine border color
+        var borderColorAttr = disabled
+            ? VdomSpectreTranslator.GetAttribute(node, "data-disabled-border-color") ?? "grey19"
+            : VdomSpectreTranslator.GetAttribute(node, "data-border-color") ?? "grey37";
+
+        // Determine border style
+        var borderStyleAttr = VdomSpectreTranslator.GetAttribute(node, "data-border-style");
+        var borderStyle = ResolveBorderStyle(borderStyleAttr);
+
+        // Create panel
+        var panel = new Panel(new Markup(markupText))
```

Comment:
the select panel should contain all options, and selected option should either be highlighted or mark with a special mark.

E.G

```
   option a
> option b (selected)
   [red]option c[/] (current option, highlighted, but not selected)
```